lib/simplecov/version.rb

```diff
@@ -1,22 +1,25 @@
 module SimpleCov
-  VERSION = "0.11.1"
-  def VERSION.to_a
+  version = "0.11.1"
```

Comment:
Oh wow, there are the most amazing bugs from the simplest things. 

I'm somewhat torn on this one because (see https://github.com/colszowka/simplecov/issues/458) we basically shouldn't be spending time on supporting rubies that were [EOLd a year ago](https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/) but this is such a relatively small thing to fix and make things work on old rubies again. Maybe simplecov should even outright refuse to be compatible with outdated rubies to make people aware of the fact they're exposing themselves to unfixed security and performance problems.

Anyway, could you please send in a PR that disables the relevant Cop, with a comment about why it's disabled and a link to here as well as the unfreezing of the version string? Also, please upgrade your Ruby, in my experience the upgrades from 1.9 on are very easy.